### ISSUE TITLE
[4.0] WCAG 1.3.1,WCAG 3.3.2: Form elements must have labels (.choices__input--cloned[autocapitalize="off"][spellcheck="false"])

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#24766]
Tags Fields

#### Issue
`Form elements must have labels` ([label](https://dequeuniversity.com/rules/axe/3.2/label?application=msftAI))
#### Target application
[Articles: New - Thursday Alpha 8 - Administration](http://localhost/joomla-cms/administrator/index.php?option=com_content&view=article&layout=edit)
#### Element path
.choices__inner > .choices__input--cloned[autocapitalize="off"][spellcheck="false"]
#### Snippet
`<input type="text" class="choices__input choices__input--cloned" autocomplete="off" autocapitalize="off" spellcheck="false" role="textbox" aria-autocomplete="list" placeholder="Type or select some options" style="width: 191px;">`
#### How to fix
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit

**Comment by brianteeman** [comment]
The problem is that the label we have is "for" an "id" that is hidden. The id needs to be removed from the hidden select element and added to the choices generated input element. 

![image](https://user-images.githubusercontent.com/1296369/56977121-695f3d00-6b6c-11e9-9352-a2064eb7306b.png)

**Comment by brianteeman** [comment]
On further investigation the problem is not coming from our implementation of choices as their own demo has the exact same problem

Possible solutions
- https://github.com/jshjohnson/Choices/pull/122
- https://github.com/jshjohnson/Choices/issues/296

**Comment by brianteeman** [comment]
This has been resolved by an upstream update of choices.js and applied with #27093
